### ISSUE TITLE
DT 3.6.0.1 Nomenclature Information

### ISSUE THREAD

**Issue by Dave22152** [darktable-org/darktable#9461]
When I downloaded version 3.6.0.1 this afternoon I saw the version number is identified as 3.6.0.  Earlier versions were always identified at the full indenture level (ie. 3.4.1.1 vs 3.4).  Granted this is a minor point, but it could create confusion among users who want to ensure they've downloaded the latest.

I placed this under the documentation category, as I don't believe this qualifies as a bug, at least in the usual sense.  

I've attached screen shots of the version that I downloaded, the Github asset listing that shows 3.6.0.1, my downloaded .exe with the correction version, then the application properties, and finally the DT About window that identifies the version as 3.6.0.

**Comment by wpferguson** [comment]
When we repackaged we were in scramble mode since the packager wasn't available and I ended up doing it.  I just repackaged the 3.6.0 release and I think Pascal renamed the installer to 3.6.0.1 so we could tell the difference, otherwise we would have had 3.6.0 that worked and 3.6.0 that didn't.  The problem way would have been to create an additional tag, release-3.6.0.1, check that out, then package it.  Now that we've learned the hard way, we'll be a little smarter next time.

**Comment by wpferguson** [comment]
Just to make my above comment clear, no code changes were made.  The error was in the packaging so the installed version, code wise, is 3.6.0.

**Comment by TurboGit** [comment]
Exactly. And a change of packaging does not require a change of software version. Maybe instead of 3.6.0.1 I should have named the installer 3.6.0-1.

**Comment by wpferguson** [comment]
Considering how many times I built the windows package last night, trying to figure out cmake and the mo files, I should have packaging down by the next time I have to do it for real :smile:

**Comment by parafin** [comment]
For macOS builds I use 4th version part as number of commits since tag. So 3.6.0.3 is 3 commits after 3.6 tag. In interface it shows up like git build, smth like 3.6.0+3~abcdefg. Even if it’s just packaging difference outside of cmake files you could just commit changes to BUILD.txt instructions file so that version gets updated. Having 2 builds that show up as the same version in interface is IMHO suboptimal regarding bug reports and such.

**Comment by wpferguson** [comment]
The way that the windows packaging works, it would be simpler to just tag it, check out the tag, then build since the packaging process picks it up.  I'll test that and see how it works later today

**Comment by Dave22152** [comment]
I don’t think anyone’s going to lose sleep over this one. After I installed the update, it just took me back for a minute like, “ … hold on… did I just reinstall the original version?”

Maybe just a short note on PIXLS is enough to close the loop.

**Comment by wpferguson** [comment]
I checked out release-3.6.0.  I added the tag release-3.6.0-1.  I checked out master.  Then I checked out release-3.6.0-1.  I did a git status and it showed I was on release-3.6.0-1.  I built and the package was darktable-3.6.0-win64.exe so I think the build picks up the first tag.  It appears that adding an additional tag does not work for building windows packages with a different release number.

Perhaps we document somewhere that we use -x to specify same code, different package and .x in the fourth position to specify different code. Or, we do as @parafin suggested and make a trivial documentation change, then tag it.
